ecommerce/serializers.py

```diff
@@ -607,7 +607,9 @@ def _validate_runs(self, run_ids, product):
         if len(product_run_course_map) < len(run_ids):
             missing_run_ids = set(run_ids) - set(product_run_course_map.keys())
             raise ValidationError(
-                {"runs": f"Unable to find run(s) with id(s) {missing_run_ids}"}
+                {
+                    "runs": f"Unable to find course run {product.content_object.text_id}, associated with product {missing_run_ids}"
```

Comment:
Related to the comment above. Please change this text to make it more relevant to actual users.

```suggestion
                    "runs": "Some invalid courses were selected."
```

Reply:
I did exactly what was mentioned on the ticket, but changes it as you suggested.